app/workers/user_subscriptions/sync_counter_cache.rb

```diff
@@ -0,0 +1,11 @@
+module UserSubscriptions
+  class SyncCounterCache
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :default, retry: 10
+
+    def perform
```

Comment:
I see this triggered in the specs below but where in the production running code do we trigger this to run?

Reply:
Ah, I left out the `config/schedule.yml` update. Updated!